.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 7):  

![image](https://github.com/Swarzstein/final-capstone/assets/57421823/2dfb1382-9526-4bf7-bd19-691fe3a236a0)


![image](https://github.com/Swarzstein/final-capstone/assets/57421823/020c9792-25b8-4dfd-a1f1-e2c462f0c96a)


That way, your code will meet the latest standards. 💪